File: tests/fixtures/openai-realtime/openai-realtime-avoid-dated-preview-snapshots-broken/01-template-literal-url.ts
import WebSocket from 'ws';

function setup(apiKey: string) {
  const url = `wss://api.openai.com/v1/realtime?model=gpt-4o-realtime-preview-2024-10-01`;
  return new WebSocket(url, { headers: { Authorization: `Bearer ${apiKey}` } });
}

File: tests/fixtures/openai-realtime/openai-realtime-avoid-dated-preview-snapshots-broken/01_template_literal_url.py
import websockets


def setup(api_key):
    ws_url = f"wss://api.openai.com/v1/realtime?model=gpt-4o-realtime-preview-2024-10-01"
    return websockets.connect(ws_url, additional_headers={"Authorization": f"Bearer {api_key}"})
